numpy/distutils/command/autodist.py

```diff
@@ -69,7 +69,10 @@ def check_gcc_function_attribute(cmd, attribute, name):
         #pragma GCC diagnostic error "-Wattributes"
         #pragma clang diagnostic error "-Wattributes"
 
-        int %s %s(void*);
+        int %s %s(void*)
```

Comment:
FWIW I am not able to reproduce this on 7.3.0 or gcc-9 versions. but if this function is not behaving as its supposed to for certain compiler versions, its certainly worth fixing.

Reply:
that's odd, This build error was perfectly reproducible on bionic with gcc-7.5 and gcc-9.2.

Reply:
@r-devulap , not sure whats going on. my gcc version for gcc-9 is gcc 9.3.0

Reply:
I was going to say, it could be Windows-specific.  But maybe not.